src/main/java/org/socialhistoryservices/delivery/reservation/service/ArchiveReservationPrintable.java

```diff
@@ -41,7 +41,8 @@ protected void draw(DrawInfo drawInfo) {
         drawNewLine(drawInfo);
 
         drawArchive(drawInfo);
-        drawInventoryNumber(drawInfo);
+        drawContainer(drawInfo);
+        //drawInventoryNumber(drawInfo);
```

Comment:
There will not always be a container, so it should draw the container info only if there is container info. Otherwise it should draw the inventory number.